contracts/Crowdfunding.sol

```diff
@@ -158,8 +173,27 @@ contract Crowdfunding {
         //emit action
     }
 
+    function refund(uint256 _campaignId, address _to) internal {
+        require(campaigns[_campaignId].state == State.OPEN, "Campaign must be open to request refund");
```

Comment:
This is questionable, and we should revisit the conditions when a campaign can be refunded